lefthook.yml

```diff
@@ -0,0 +1,5 @@
+pre-commit:
+  commands:
+    js:
+      glob: "*.{js,jsx}"
+      run: pretty-quick --staged && pnpm test
```

Comment:
I wanted to try `lefthook` for a while. This seems like a pretty nice alternative to husky.